### PR TITLE
Revert "Revert "PENV-3: fix jepsens""

### PR DESCRIPTION
**Pull request by Vany**
Reverts insolar/insolar-jepsen#141

### CODE REVIEW

File: run-test.py [hunk 1]
```diff
@@ -838,10 +841,10 @@ def deploy_observer(path, keep_database=False):
            "/../observer", flags="-r", ignore_errors=True)
     ssh(OBSERVER, "cd "+INSPATH +
         "/../observer && GO111MODULE=on make all && mkdir -p .artifacts")
-    scp_to(OBSERVER, "/tmp/insolar-jepsen-configs/observer.yaml",
-           INSPATH+"/../observer/.artifacts/observer.yaml")
-    scp_to(OBSERVER, "/tmp/insolar-jepsen-configs/observerapi.yaml",
-           INSPATH+"/../observer/.artifacts/observerapi.yaml")
+
+    for сqw in "".split("observer observerapi stats-collector migrate"):
```
Comment (afiskon) [inline run-test.py:845]: This line doesn't do what you expect.

Comment (Vany) [reply run-test.py:845]: explain pls

Comment (afiskon) [reply run-test.py:845]:
```
$ bpython
>>> "".split("observer observerapi stats-collector migrate")
['']
```

Also please avoid using Perl idioms in Python code. Just use regular lists.

Comment (Vany) [reply run-test.py:845]: thx

**Pull request closed without merging**